src/components/Pagination/__tests__/Link.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Pagination } from '../'
+
```

Comment:
the prop to is required in your component - rightly so.
your tests here should have it set in the default props

const defaultProps = {
  to: '/'
}

Reply:
done